### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by pdion891**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Tested with Terraform version : 0.9.5
 # Installation
 
 1. Download the cloud.ca Terraform provider binary for your OS from the [releases page](https://github.com/cloud-ca/terraform-provider-cloudca/releases).
-2. Copy the provider to the directory where Terraform is located.
+2. Copy the provider to the plugin directory `~/.terraform.d/plugins`.
```
Comment (pdube) [inline README.md]: Which version of terraform are you using?

File: cloudca/README.md [hunk 2]
```diff
@@ -175,7 +175,7 @@ The following arguments are supported:
 - user_data - (Optional) User data to add to the instance
 - ssh_key_name - (Optional) Name of the SSH key pair to attach to the instance. Mutually exclusive with public_key.
 - public_key - (Optional) Public key to attach to the instance. Mutually exclusive with ssh_key_name.
-- root_size_in_gb - (Optional) Size of the root volume of the instance. This only works for templates that allows root volume resize.
+- root_volume_size_in_gb - (Optional) Size of the root volume of the instance. This only works for templates that allows root volume resize.
```
Comment (pdube) [inline cloudca/README.md]: Good catch!

File: README.md (hunk 1 above)
Comment (pdion891) [reply README.md]: `Terraform v0.11.1`

Comment (pdube) [reply README.md]: If all is working as expected, we could also bump the Tested with v0.9.5 version on the README

**Pull request merged**